pyuvsim/telescope.py

```diff
@@ -45,6 +45,15 @@ class BeamList:
     uvb_params : dict (optional)
         Options to set uvb_params, overriding settings from passed-in UVBeam objects.
 
+    check
+        Whether to perform a consistency check on the beams (i.e. asserting that several
+        of their defining parameters are the same for all beams in the list).
+
+    force_check
+        The consistency check is only possible for object-beams (not string mode). If
+        `force_check` is True, it will convert beams to object-mode to force the check
+        to run (then convert back to string mode).
+
```

Comment:
There should not be empty lines between these items in the Parameters section. We also usually indicate type here, but maybe that's not required because it's done in the function signature?

Reply:
Looking at the RTD docs, I think we should add the types here because they don't get rendered the same way even though they are in the signature.

<img width="688" alt="Screen Shot 2022-02-01 at 11 58 30 AM" src="https://user-images.githubusercontent.com/3084869/152041936-e809f85c-a405-43c2-8775-a8f785790c33.png">